modules/ui/src/Form/StockTransactions1.php

```diff
@@ -43,14 +43,6 @@ public function buildForm(array $form, FormStateInterface $form_state) {
   /**
    * {@inheritdoc}
    */
-  public function validateForm(array &$form, FormStateInterface $form_state) {
```

Comment:
Maybe use the phpcs.xml Commerce uses https://github.com/drupalcommerce/commerce/blob/8.x-2.x/phpcs.xml